internal/activities/validate_file_formats.go

```diff
@@ -96,3 +86,34 @@ func (a *ValidateFileFormats) Execute(
 
 	return &ValidateFileFormatsResult{Failures: failures}, nil
 }
+
+func parseFormatList(r io.Reader) (formatList, error) {
+	var i int
+	formats := make(formatList)
+
+	cr := csv.NewReader(r)
+	for {
+		row, err := cr.Read()
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			return nil, fmt.Errorf("invalid CSV: %v", err)
+		}
+
+		// Skip the first row.
+		if i > 0 {
```

Comment:
The default configuration above doesn't have a header, should we remove this skip or expect a header in the CSV?

Reply:
On the other hand, the configuration from the testdata folder includes the header.

Reply:
Oh, that's a bug, the  default config should have a header.

Reply:
Fixed.